pybamm/models/full_battery_models/lead_acid/full.py

```diff
@@ -6,7 +6,7 @@
 
 
 class Full(BaseModel):
-    """Porous electrode model for lead-acid, from [1]_, based on the Full
+    """Porous electrode model for lead-acid, from [1]_, based on the Newman-Tiedemann
```

Comment:
Minor typo: should the underscore be removed?

Reply:
this is for sphinx cross-ref